packages/FormElement/stories/FormElement.stories.mdx

```diff
@@ -0,0 +1,265 @@
+/>
+
+<Description markdown={Readme} />
+
+## More Examples
+
+### Index
+
+- <a href="#accessibility">Accessibility</a>
+- <a href="#alternate-layouts">Alternate Layout</a>
+
+  - <a href="#checkboxes">Checkboxes</a>
+  - <a href="#radios">Radios</a>
+  - <a href="#date-picker">Date Picker</a>
+  - <a href="#listbox">ListBox</a>
+  - <a href="#buttongroup">ButtonGroup</a>
+  - <a href="#nested-formelement">Nested FormElements</a>
+
+### Accessibility
+
+The Formelement has been designed with accessibiklity in mind. By using `<FormElement.Instructions>` users can add instructions for the end user. Users can also make use of the `<FormElement.Description>` to add additional direction for consumers.
+
+```jsx
+<FormElement label="Form Label">
+  <FormElement.Instructions>
+    <span>
+      Example text for extra panel for questionnaires. Example text for extra panel for questionnaires Example text for
+      extra panel for questionnaires. Example text for extra panel for questionnaires
+    </span>
+  </FormElement.Instructions>
+  <FormElement.Content>
+    {({ idForLabel, ariaDescribedBy }) => (
+      <Input
+        id={idForLabel}
+        onChange={handleChange}
+        value={value}
+        placeholder="Form placeholder"
+        aria-describedby={ariaDescribedBy}
+        hasError={Boolean(errorText.length)}
+      />
+    )}
+  </FormElement.Content>
+  <FormElement.Description>
+    <span>This is description text</span>
+  </FormElement.Description>
+  <FormElement.Error>{errorText}</FormElement.Error>
+  <FormElement.Help>
+    Give me some help. <a href="wegalvanize.com">Learn more</a>.
+  </FormElement.Help>
+</FormElement>
+```
+
+##### Result
+
+<FormElementStory>
+  <AccessibilityExample />
+</FormElementStory>
+
+### Alternate Layouts
+
+The FormElement can be used in a variety of formats. These can include FormElements with checkboxes, radios, listbox's etc.
+
+#### Checkboxes
+
+```jsx
+<FormElement hasFieldSet label="Form Label">
+  <FormElement.Content>
+    {({ ariaDescribedBy: ariaDescribedByOuter }) => getCheckboxOptions(ariaDescribedByOuter)}
+  </FormElement.Content>
+  <FormElement.Description>
+    <span>Description of this field.</span>
+  </FormElement.Description>
+  <FormElement.Error>{errorText}</FormElement.Error>
+  <FormElement.Help>
+    Give me some help. <a href="wegalvanize.com">Learn more</a>.
+  </FormElement.Help>
+</FormElement>
+```
+
+##### Result
+
+<CheckboxExample />
+
+#### Radios
+
+```jsx
+<FormElement hasFieldSet label="Form Label">
+  <FormElement.Content>
+    {({ ariaDescribedBy: ariaDescribedByOuter }) => getRadioOptions(ariaDescribedByOuter)}
+  </FormElement.Content>
+  <FormElement.Description>
+    <span>Description of this field.</span>
+  </FormElement.Description>
+  <FormElement.Error>{errorText}</FormElement.Error>
+  <FormElement.Help>
+    Give me some help. <a href="wegalvanize.com">Learn more</a>.
+  </FormElement.Help>
+</FormElement>
+```
```

Comment:
This is basically the same pattern as the checkboxes.  Maybe we don't need the code snippet, we can just _explain_ that they're basically the same and show both results.  Let's keep them as separate Examples stories though, I like that... but for the DOCS page, maybe less is more (and less code to maintain if things change).